src/react_bird.cpp

```diff
@@ -248,13 +248,13 @@ void ReactBird::init()
 
     double c1 = MY_PIS*epsilon*r->coeff[2]/(2.0*sigma) *
       sqrt(mr/(2.0*update->boltz*tref)) *
-      pow(tref,1.0-omega)/pow(update->boltz,r->coeff[3]-1.0+omega) *
-      tgamma(z+2.5-omega) / MAX(1.0e-6,tgamma(z+r->coeff[3]+1.5));
+      pow(tref,1.0-omega)/pow(update->boltz,r->coeff[3]-1.0+omega); // *
+//      tgamma(z+2.5-omega) / MAX(1.0e-6,tgamma(z+r->coeff[3]+1.5));
```

Comment:
@aborner1 what is the purpose of this change? Seems unrelated to computing chemistry rates with performing reactions?

Reply:
"z" is now variable and needs to be computed on a per reaction basis in react_tce as opposed to a priori in react_bird, hence why it got moved there.

Reply:
I see. Any ideas why the `ambi` test would fail?

Reply:
Yes. I removed the  "MAX(1.0e-6,..." part, because I don't think it should be there.
If you add it back at line 154, the test will pass.

Now, as to why I did that.
This was a fix from many years ago, when we realized that the gamma function would overflow if its argument was negative. In my opinion, I would prefer that either a warning is issued to the user, or the code stops running, when the temperature exponent of a certain reaction is out of range (i.e., too negative), as opposite to silently run (which is the case now), the user not be made aware.
The O and N ionization reactions (last 2 in tce file) have temperature exponents that should not be allowed in the TCE model because they lead to negative gamma function arguments, so they need to be refitted to have a temperature exponent close to 0.

This is more philosophical than anything else.
So I think the short version is, if you want the reg test to pass, add the "MAX" part back in, but I still think the whole "temperature exponent" check should eventually be addressed, but I guess it should be its own PR.